src/providers/definition/common.ts

```diff
@@ -71,12 +71,12 @@ export const viewSuggestions = [
 		}
 	},
 	{ // handler
-		regExp: /on(.*?)=["']$/,
+		regExp: /on(.*?)=["'][A-Za-z]*$/,
 		files () {
-			return [ related.getTargetPath('js') ];
+			return [ related.getTargetPath('js')];
 		},
 		definitionRegExp (text) {
-			return new RegExp(`function ${text}\\s*?\\(`);
+			return new RegExp(`function ${text}s*?\\(`);
```

Comment:
I think we need to keep the `\\s` here as without it a function like the below will fail to pick up (note the space before the first parenthesis)

```js
function clearLogs () {
	logger.clearHistory();
	showLogs();
}
```